compiler/src/dmd/cparse.d

```diff
@@ -1906,6 +1906,8 @@ final class CParser(AST) : Parser!AST
                     {
                         auto str = asmName.peekString();
                         p.mangleOverride = str;
+//                      p.adFlags |= AST.VarDeclaration.nounderscore;
+                        p.adFlags |= 4; // cannot get above line to compile on Ubuntu
```

Comment:
That's weird

Reply:
I know. I got weird errors from the test suite from line 1909 that I could not reproduce. My plan is, after this is pulled, to do a PR that uses line 1909 instead. That way, the problem won't delay this PR which is blocking people.